vllm/core/scheduler.py

```diff
@@ -347,7 +382,11 @@ def _schedule(self) -> SchedulerOutputs:
             for seq_group in self.running)
 
         scheduler_outputs = SchedulerOutputs(
-            scheduled_seq_groups=self.running,
+            scheduled_seq_groups=[
+                ScheduledSequenceGroup(seq_group=running_group,
+                                       token_chunk_size=1)
```

Comment:
is it always 1 for decode? if so let's document it in the `ScheduledSequenceGroup` (actually change it to dataclass for easier documentation?)